app/views/repositories/_repository_header.html.erb

```diff
@@ -0,0 +1,88 @@
+<%#-- copyright
+OpenProject is a project management system.
+Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See doc/COPYRIGHT.rdoc for more details.
+
+++#%>
+
+<% content_for :header_tags do %>
+  <%= javascript_include_tag 'repository_navigation' %>
+<% end %>
+<%= toolbar title: l('repositories.named_repository',
+                     vendor_name: @repository.class.vendor_name) do %>
+  <% if @instructions && @instructions.available? %>
+  <li class="toolbar-item toolbar-input-group">
+    <div class="toolbar-input-group--affix -prepend">
+      <span><%= @instructions.checkout_command %></span>
+    </div>
+    <input id="repository-checkout-url"
+           type="text" class="-clickable" size="40"
+           value="<%= @instructions.checkout_url %>"
+           readonly
+           copy-to-clipboard>
+    <button class="toolbar-input--affix toolbar-input-group--affix -append"
+            title="<%= l(:label_copy_to_clipboard) %>"
```

Comment:
Not sure if it is enough to set the title only and not including a `<span class="hidden-for-sighted">some text</span>`.

ChromeVox seems to read it out correctly, but I can't test Jikes and don't know our exact guidelines here...